### PR TITLE
Add R package PLNmodels

### PR DESCRIPTION
**Pull request by cpauvert**
Hi, 
Here is a PR that suggest the CRAN R package [`PLNmodels`](https://github.com/jchiquet/PLNmodels).

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/8078f7fd66080908ec4244a5d6d27b4765a3e60f/recipes/r-plnmodels/meta.yaml#L91))
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://github.com/conda-forge/staged-recipes/blob/8078f7fd66080908ec4244a5d6d27b4765a3e60f/recipes/r-plnmodels/meta.yaml#L12))
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

The package was tested locally and I also tested the creation of an environment with the newly built package. Both ran smoothly.
Thanks in advance for the reviews and suggestions.
Best, 
Charlie

### CODE REVIEW

**Comment by cpauvert** [comment]
@jchiquet Could you please comment here to confirm you are ok with being listed as a maintainer?

**Comment by jchiquet** [comment]
> @jchiquet Could you please comment here to confirm you are ok with being listed as a maintainer?

I confirm that I agree for being listed as maintainer of PLNmodels

**Comment by cpauvert** [comment]
The test failed with 9e84330 and [logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=232806&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=862) suggest a missing dependency with `libopenblas`, even if the dependency isn't visible in PLNmodels! Nonetheless `libopenblas` happen to be available as a [conda package](https://anaconda.org/anaconda/libopenblas) which is great but including it in the `meta.yaml` (3aeff78) resulted again in a failed test due to [unsatisfable dependencies](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=232881&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=993).
So I'm puzzled right now

File: recipes/r-plnmodels/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    - r-corrplot
+    - r-dplyr
+    - r-ggplot2
+    - r-glassofast
+    - r-gridextra
+    - r-igraph
+    - r-magrittr
+    - r-nloptr >=1.2.0
+    - r-tidyr
+    - libopenblas
```
Comment (dbast) [inline recipes/r-plnmodels/meta.yaml:56]: Please change that to `libblas` to allow switching https://conda-forge.org/docs/maintainer/knowledge_base.html#switching-blas-implementation

**Comment by dbast** [comment]
Can you use https://github.com/bgruening/conda_r_skeleton_helper to update the PR by committing the exact output... that cleans up comments, brings back the cran meta data for easier review etc.

File: recipes/r-plnmodels/meta.yaml (hunk 1 above)
Comment (cpauvert) [reply recipes/r-plnmodels/meta.yaml:56]: Thank you @dbast for your review and suggestion. I'll do that!

**Comment by cpauvert** [comment]
> Can you use https://github.com/bgruening/conda_r_skeleton_helper to update the PR by committing the exact output... that cleans up comments, brings back the cran meta data for easier review etc.

Thanks @dbast. Done in the last commit. Hope the tests goes well.

**Comment by dbast** [comment]
Requires r-glassofast to be added to conda-forge

**Comment by cpauvert** [comment]
> Requires r-glassofast to be added to conda-forge

Ok thank you @dbast. I wil submit this one before then.

**Comment by cpauvert** [comment]
Hum I am puzzled. `r-glassofast` exists on the `r` channel: [here](https://anaconda.org/r/r-glassofast). That's why this dependency was correctly filled locally for me. Is this because of the "Avoid external dependencies" clause on the [documentation](https://conda-forge.org/docs/maintainer/adding_pkgs.html#avoid-external-dependencies) that the Azure bot is not able to find  `r-glassofast` on the `r` channel?
Best, 
Charlie

**Comment by dbast** [comment]
The r channel only provides R 3.6 packages... conda-forge builds packages for R 3.6 and R 4.0.

**Comment by cpauvert** [comment]
> The r channel only provides R 3.6 packages... conda-forge builds packages for R 3.6 and R 4.0.

Ok thank you very much for the explanation @dbast

**Comment by cpauvert** [comment]
Okay, now `r-glassofast` is now available for both R3.6 and R4.0 (thanks to the quick merge of @dbast).
```
# Name                       Version           Build  Channel             
r-glassofast                     1.0   r36h13aaf81_0  conda-forge         
r-glassofast                     1.0   r36ha65eedd_0  pkgs/r              
r-glassofast                     1.0   r40h13aaf81_0  conda-forge 
```
I'll trigger the test again.
Best, 
Charlie

**Comment by dbast** [comment]
Thanks!

**Comment by cpauvert** [comment]
Thank you @dbast for the quick merge!

**Comment by cpauvert** [comment]
Hi @dbast  sorry for the question but is there an issue with automatic feedstock creation (due to recent changes 15c688b638d831b9c3c63dd033071b9a8462b53e)? I noticed that recent recipes that were merged did not make it to the feedstocks (#13065, #13062  , #13063 and this one) despite the usual amount of time needed for the bot.
Best, 
Charlie

**Comment by cpauvert** [comment]
Actually the repo was created (https://github.com/conda-forge/r-plnmodels-feedstock) but it is empty

**Pull request merged**